.github/workflows/build-and-deploy.yml

```diff
@@ -38,6 +38,26 @@ jobs:
           name: build-artifacts
           path: dist
 
+  formatting:
+    runs-on: ubuntu-24.04
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 18
+
+      - name: Run install
+        uses: borales/actions-yarn@v5
+        with:
+          cmd: install
+
+      - name: Check Formatting
+        uses: borales/actions-yarn@v5
+        with:
+          cmd: format:check
```

Comment:
@sharunkumar should we use `format:write` instead?

Reply:
I think `format:check` makes sense indeed, because we want that information to be communicated back to the maintainers rather than fix automatically. [One example of such a fail](https://github.com/arunsathiya/portfolio/actions/runs/10539625165), where it [prompted me to fix further](https://github.com/arunsathiya/portfolio/commit/c370b6f346144f6f9f89979bcd4b9304ad681cb1).